code/game/objects/items/devices/flashlight.dm

```diff
@@ -415,7 +415,7 @@
 /obj/item/flashlight/flare/Initialize(mapload)
 	. = ..()
 	if(randomize_fuel)
-		fuel = rand(25 MINUTES, 35 MINUTES)
+		fuel = rand(5 MINUTES, 10 MINUTES)
```

Comment:
I think they should be a bit more consistent, the range is too wide here. Also, it should at least burn twice as bright if it were to last less than half as long.

Reply:
I'm changing the lighting w https://github.com/tgstation/tgstation/pull/81425 which makes this a bit harder.
"twice as bright" isn't really meaningful at some point cause we don't have bloom, so we just end up blowing out the color on the light.

I will see about the range tho